.github/workflows/pr-tests.yml

```
@@ -0,0 +1,204 @@
+    
+    steps:
+    - uses: actions/labeler@v4
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+    
+    - name: Add status labels
+      uses: actions/github-script@v7
+      with:
+        script: |
+          const labels = [];
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)